packages/zendesk-adapter/src/group_change.ts

```diff
@@ -55,8 +57,16 @@ const recurseIntoInstanceChangeToGroupId: ChangeIdFunction = async change => {
 const typeNameChangeGroupId: ChangeIdFunction = async change =>
   getChangeData(change).elemID.typeName
 
+// sections need to be grouped separately as there are dependencies with 'parent_section_id'
+const SectionChangeGroupId: ChangeIdFunction = async change =>
```

Comment:
nit: start lowecase for "variable" names
```suggestion
const sectionChangeGroupId: ChangeIdFunction = async change =>
```